### PR TITLE
DTFS2-5464 - Portal Bug - Portal facilities issued stage

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue: 

* Facilities in dashboard facilities table were showing issued when not yet submitted to UKEF

# Changes made: 

* Added hasBeenIssuedAndAcknowledged flag to logic so when that is set to true, then issued shows
* Added e2e test
* Added mapping for this extra flag in portal-api query

### CODE REVIEW

File: portal/templates/dashboard/_macros/dashboard-facilities-table.njk [hunk 1]
```diff
@@ -25,7 +25,7 @@
       {% endif %}
     {% endset %}
 
-    {% if facility.hasBeenIssued %}
+    {% if facility.hasBeenIssued and facility.hasBeenIssuedAndAcknowledged %}
```
Comment (ttbarnes) [inline portal/templates/dashboard/_macros/dashboard-facilities-table.njk:28]: Are we sure we want this - last week had a requirement change where an Issued facility should show as Issued on the BSS deal page, before it submitted to UKEF?

Comment (Zainzzkk) [reply portal/templates/dashboard/_macros/dashboard-facilities-table.njk:28]: @IgnatG

File: e2e-tests/portal/cypress/integration/journeys/dashboard/fixtures/MIA-deal-with-accepted-status-issued-facilities-cover-start-date-in-past.js [hunk 2]
```diff
@@ -0,0 +1,462 @@
+const now = new Date().valueOf();
+const nowMinusDay = () => {
+  const date = new Date();
+  date.setDate(date.getDate() - 1);
+  return date.valueOf();
+};
+
+const deal = {
```
Comment (ttbarnes) [inline e2e-tests/portal/cypress/integration/journeys/dashboard/fixtures/MIA-deal-with-accepted-status-issued-facilities-cover-start-date-in-past.js:8]: is there no existing mock we can use for this and extend?

File: portal/templates/dashboard/_macros/dashboard-facilities-table.njk (hunk 1 above)
Comment (IgnatG) [reply portal/templates/dashboard/_macros/dashboard-facilities-table.njk:28]: interesting!
@Zainzzkk  can you clarify with Clara?

**Pull request closed without merging**